Exporter/src/main/java/uk/ac/ebi/biostd/exporter/schedulers/ScheduledTasksConfiguration.java

```diff
@@ -58,38 +60,82 @@ public ScheduledTasksConfiguration(
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "jobs.dummy", name="enabled", havingValue="true")
+    @ConditionalOnProperty(prefix = "jobs.dummy", name = "enabled", havingValue = "true")
     public CronTask dummyScheduler() {
         return new CronTask(() -> log.info("Hi there, I'm the dummy task"), dummyCron);
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "jobs.full", name="enabled", havingValue="true")
+    @ConditionalOnProperty(prefix = "jobs.full", name = "enabled", havingValue = "true")
     public CronTask fullScheduler() {
-        return new CronTask(fullExporter::execute, fullCron);
+        return new CronTask(new FullExportJob(fullExporter::execute), fullCron);
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "jobs.partial", name="enabled", havingValue="true")
+    @ConditionalOnProperty(prefix = "jobs.partial", name = "enabled", havingValue = "true")
     public CronTask partialScheduler() {
-        return new CronTask(partialExporter::execute, partialCron);
+        return new CronTask(new PartialExportJob(partialExporter::execute), partialCron);
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "jobs.pmc", name="enabled", havingValue="true")
+    @ConditionalOnProperty(prefix = "jobs.pmc.export", name = "enabled", havingValue = "true")
     public CronTask pmcScheduler() {
-        return new CronTask(pmcExporter::execute, pmcCron);
+        return new CronTask(new PmcLinksExportJob(pmcExporter::execute), pmcCron);
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "jobs.stats", name="enabled", havingValue="true")
+    @ConditionalOnProperty(prefix = "jobs.stats", name = "enabled", havingValue = "true")
     public CronTask statsScheduler() {
-        return new CronTask(statsExporter::execute, statsCron);
+        return new CronTask(new StartsExportJob(statsExporter::execute), statsCron);
     }
 
     @Bean
     @ConditionalOnProperty(prefix = "jobs.releaser", name = "enabled", havingValue = "true")
     public CronTask releaser() {
         return new CronTask(releaserJob::execute, releaserCron);
     }
+
+    @AllArgsConstructor
+    public static class FullExportJob implements Runnable {
+
+        private final Runnable runnable;
```

Comment:
Why implement Runnable and then have a runnable inside? 🤔

Reply:
only to be able to see the name and not the lambda expression :(  really sad I know